### PR TITLE
Red Desert tiles (Trijent asset port)

### PR DESCRIPTION
**Pull request by UnicornOnLSD**
## About the PR

desert tiles
port for trijent dam
All variants of red desert tiles, including rock, grass and dirt

(I suffered.)
## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image

**Changelog**

no cl no fun

### CODE REVIEW

File: Resources/Textures/_RMC14/Tiles/planet/red_desert/desert_transition_edge8.png
Comment (QuietlyWhisper) [inline Resources/Textures/_RMC14/Tiles/planet/red_desert/desert_transition_edge8.png:1]: Empty sprite?

File: Resources/Prototypes/_RMC14/Tiles/trijent.yml
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Tiles/trijent.yml:1]: New content IDs should be `RMC` and not `CM`

File: Resources/Locale/en-US/_RMC14/tiles/tiles.ftl [hunk 1]
```diff
@@ -28,6 +28,7 @@ tiles-cm-sand = sand tile
 tiles-cm-grass = grass tile
 tiles-cm-dirt-grass = dirt grass tile
 tiles-cm-rock = rock tile
+tiles-cm-red-desert = red desert tile
```
Comment (QuietlyWhisper) [inline Resources/Locale/en-US/_RMC14/tiles/tiles.ftl:31]: `tiles-rmc-red-desert = red desert tile`
Make sure to replace in .YML file as well

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

File: Resources/Textures/_RMC14/Tiles/planet/red_desert/desert_transition_edge8.png
Comment (UnicornOnLSD) [reply Resources/Textures/_RMC14/Tiles/planet/red_desert/desert_transition_edge8.png:1]: mbad, this one slipped under the radar, will nuke it

**Review by DrSmugleaf — APPROVED** [review]

**Comment by DrSmugleaf** [comment]
Thanks for the PR!

**Pull request merged**